crates/core/tedge_actors/src/builders.rs

```diff
@@ -163,14 +164,17 @@ pub trait MessageSink<M: Message, Config> {
     ///
     /// sender.send("Hello!").await?;
     ///
-    /// let mut receiver = receiver.with_timeout(Duration::from_millis(100));
-    /// assert_eq!(receiver.recv().await, Some('H'));
-    /// assert_eq!(receiver.recv().await, Some('e'));
-    /// assert_eq!(receiver.recv().await, Some('l'));
-    /// assert_eq!(receiver.recv().await, Some('l'));
-    /// assert_eq!(receiver.recv().await, Some('o'));
-    /// assert_eq!(receiver.recv().await, Some('!'));
-    /// assert_eq!(receiver.recv().await, None);
+    /// #[cfg(feature = "test-helpers")]
```

Comment:
@jarhodes-sag @Bravo555 Is there any way to avoid repeating this `cfg` guard on each and every line that uses this feature flag like this?

Reply:
You might be able to wrap the doc test in `# fn main() {` and `# }` and then put the feature flag guard above the function definition.

Reply:
The drawback is that now the `--all-features` flag is required for `cargo doc -p tedge_actors --all-features` and for `cargo test -p tedge_actors --all-features`.

Do we have a way to tell Cargo that the `test-helpers` feature is required for `test` and `doc`? If yes, this will also help to fix the issue I have with the [examples used by the doc comments](https://github.com/thin-edge/thin-edge.io/pull/1952/files#diff-606921f32fd050a581b7c4ae163ad11d2eef50ff1a26a0338e044721cb556b01R359)